### ISSUE TITLE
Collapse large facet lists by default

### ISSUE THREAD

**Issue by Otterfan** [BCDigSchol/census#180]
If there are 100+ topic authors in a search for "Greek", only show the first ~15 and make the full list openable/collapsable.

**Comment by peter9712** [comment]
@Otterfan @GreekCommons Yes, collapsing is indeed very important for any large lists. That is, large lists of filter categories (like place names), and also large lists of components in an expanded record (as well as, perhaps, long fields, like 'Other notes').
